README.md

```diff
@@ -80,19 +90,25 @@ class AnyPageSpec
 }
 ```
 
-To run the tests do,
+To run the tests you can either do,
+
+```shell script
+sbt a11y:test  
+```
+
+Or alternatively you can run,
 
 ```shell script
-sbt a11yTest
+sbt accessibilityAssessment
```

Comment:
I'd almost just have "a11y:test" rather than introduce the new key 🤷‍♂️

Reply:
I have made the change, thank you.